src/components/MenuEnvelope.vue

```diff
@@ -0,0 +1,321 @@
+<!-- Standard Actions menu for Envelopes -->
+<template>
+	<div class="app-content-list-item-menu">
```

Comment:
I'm pretty sure this class should only be used in the *app content list* (envelope list) and no in the details on the right